File: lista-de-exercicios/exercicio-16/Livro.ts
export class Livro {
    private titulo: string;
    private autor: string;
    private emprestado: Boolean;
    
    public constructor (
        titulo: string,
        autor: string) {
        this.titulo = titulo;
        this.autor = autor;
    }

    public getAutor() {
        return this.autor;
    }

    public getTitulo() {
        return this.titulo;
    }
}

File: lista-de-exercicios-java/src/main/java/org/example/exercicio16/Livro.java
package org.example.exercicio16;

public class Livro {
    private final String titulo;
    private final String autor;

    public Livro(String titulo, String autor) {
        this.titulo = titulo;
        this.autor = autor;
    }

    public String getTitulo() {
        return titulo;
    }
}
